src/dom.js

```diff
@@ -111,6 +111,9 @@ export function closest(element, callback) {
  * @return {?Element}
  */
 export function closestByTag(element, tagName) {
+  if (element.closest) {
```

Comment:
Just tried now: http://jsperf.com/element-closest-vs-parent-walk/3 - native `closest` is a lot faster.